gtsam/discrete/DecisionTree-inl.h

```diff
@@ -549,15 +544,15 @@ namespace gtsam {
   template<typename L, typename Y>
   template<typename Iterator> DecisionTree<L, Y>::DecisionTree(
       Iterator begin, Iterator end, const L& label) {
-    root_ = compose(begin, end, label);
+    root_ = Choice::Unique(compose(begin, end, label));
```

Comment:
Why do all these have a Unique now? That was not detailed in the PR comment

Reply:
Okay I undid that. Looks like it was a remnant of debugging.